main.yml

```diff
@@ -290,6 +290,31 @@ landing:
   channel: '#flip-mode-squad'
   deployment_repo: https://github.com/RedHatInsights/landing-page-frontend-build
 
+migrations:
+  title: Migrations
+  frontend:
+    sub_apps:
+      - id: migration-analytics
+        default: true
+  top_level: true
+
+migration-analytics:
+  title: Migration Analytics
+  api:
+    versions:
+      - v1
+    alias:
+      - xavier
+    tags:
+      - value: "experimental"
```

Comment:
Tags currently does not have any effect, but good point in adding these! You can keep these in here, it will just have no effect, do you want to somehow show in the UI that users might experience some problems when accessing this API?

Reply:
let's keep it since it will have no effect... showing messages like `users might experience some problems when accessing this API` were not in our plans so I think it is OK for now not to show such message

Reply:
This value is fine for this PR. 
Looking ahead - can you provide a list of the other permissible values please so we can decide which tag is most appropriate for Migration Analytics?

Reply:
Currently we are not showing any tags in api docs, but there were plans to show them. We were planning to show just the value and title so users can filter the table in docs api.